src/components/SecondaryPanes/Frames/index.js

```diff
@@ -8,32 +8,23 @@ import PropTypes from "prop-types";
 import React, { Component } from "react";
 import { bindActionCreators } from "redux";
 import { connect } from "react-redux";
-import { createSelector } from "reselect";
 
-import { get } from "lodash";
 import type { Frame } from "debugger-html";
-import type { SourcesMap } from "../../../reducers/sources";
 
 import FrameComponent from "./Frame";
 import Group from "./Group";
 
 import renderWhyPaused from "./WhyPaused";
 
 import actions from "../../../actions";
-import {
-  annotateFrame,
-  collapseFrames,
-  formatCopyName
-} from "../../../utils/frame";
+import { collapseFrames, formatCopyName } from "../../../utils/frame";
 import { copyToTheClipboard } from "../../../utils/clipboard";
 
 import {
-  getFrames,
   getFrameworkGroupingState,
   getSelectedFrame,
-  getSourceInSources,
-  getSources,
-  getPause
+  getPause,
```

Comment:
I love seeing this complexity go away. Especially lodash, reselect, the selectors...